### PR TITLE
Node 10

### PR DESCRIPTION
**Pull request by diasdavid**
Let's give Node 10 a spin! :)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -110,7 +110,7 @@
     "ipfs-block": "~0.7.1",
     "ipfs-block-service": "~0.14.0",
     "ipfs-multipart": "~0.1.0",
-    "ipfs-repo": "~0.20.0",
+    "ipfs-repo": "~0.22.0",
```
Comment (diasdavid) [inline package.json]: @jacobheun CI is not happy with the latest ipfs-repo. How did you run the tests before (ref: https://github.com/ipfs/js-ipfs-repo/pull/168#issuecomment-392808793)

Comment (jacobheun) [reply package.json]: This is due to the ipld PR, https://github.com/ipld/js-ipld/pull/129, that needs released. I will update https://github.com/ipfs/js-ipfs/pull/1358 once that is done.

The current versioning scheme results in old leveldown being installed for ipld, which breaks the  install.

Comment (jacobheun) [reply package.json]: @diasdavid I already have a PR open for node 10 that is running tests now, https://github.com/ipfs/js-ipfs/pull/1358. It also includes the fix for the breaking change in ipfs-repo 0.20.1 https://github.com/ipfs/js-ipfs-repo/issues/165, which is the errors your PR is getting.

Comment (diasdavid) [reply package.json]: Missed that one. I had read from your message that you had tested against js-ipfs master. All good then :)

**Pull request closed without merging**